packages/components/pie-notification/src/index.ts

```diff
@@ -84,19 +81,9 @@ export class PieNotification extends LitElement implements NotificationProps {
     @state()
     protected _hasIconClass = false;
 
-    @state()
-    protected _hasContentGutter = false;
-
     // Renders a `CSSResult` generated from SCSS by Vite
     static styles = unsafeCSS(styles);
 
-    /**
-     * Lifecycle method executed after component renders.
-     */
-    protected firstUpdated (): void {
```

Comment:
`firstUpdated` doesn't run on the server, thus, resulting in the following error in nextjs: 

![image](https://github.com/justeattakeaway/pie/assets/28605803/4b1e5c29-6c83-4450-9c8e-f160d62fdb85)


we already use `willUpdate` on line 92 to trigger the same behavior and the cycle runs in the server too so it should be enough.